docs/developer/how-to/build-docs.rst

```diff
@@ -0,0 +1,38 @@
+Build the docs using sphinx
+===========================
+
+You can build the `sphinx`_ based docs from the project directory by running::
+
+    $ tox -e docs
+
+This will build the static docs on the ``docs`` directory, which includes API
+docs that pull in docstrings from the code.
+
+.. seealso::
+
+    `documentation_standards`
+
+The docs will be built into the ``build/html`` directory, and can be opened
+locally with a web browser::
+
+    $ firefox build/html/index.html
+
+Autobuild
+---------
+
+You can also run an autobuild process, which will watch your ``docs``
+directory for changes and rebuild whenever it sees changes, reloading any
+browsers watching the pages::
+
+    $ tox -e docs autobuild
+
+You can view the pages at localhost::
+
+    $ firefox http://localhost:8000
+
+If you are making changes to source code too, you can tell it to watch
+changes in this directory too::
+
+    $ tox -e docs autobuild -- --watch event_model/tests
```

Comment:
Here, I am still not sure if we need to watch the `event_model/` dir or the `event_model/tests` dir. I guess a developer may want to rebuild the docs if any module in the library is changed, not only the test ones.

Reply:
I think you're right @mrakitin, it should be the event model dir

```suggestion
    $ tox -e docs autobuild -- --watch event_model/
```